scripts/ci/unit_test.sh

```diff
@@ -26,10 +27,11 @@ report_coverage_travis() {
 }
 
 echo Running tests:
-# tests with negated `unittests` build tag will not be ran
-go test -tags=unittests -covermode=count -coverprofile=$COVER_PROFILE.tmp ./apis/... ./controllers/... ./pkg/...
-< $COVER_PROFILE.tmp grep -v "zz_generated" > $COVER_PROFILE
-
+# tests with negated `unittests` build tag will not be run
+go test -tags=unittests -covermode=atomic -coverprofile="$COVER_PROFILE".tmp -p "$COV_THREAD_COUNT" ./apis/... ./controllers/... ./pkg/...
+# Remove generated files from coverage profile
+grep -v "zz_generated" "${COVER_PROFILE}.tmp" > "${COVER_PROFILE}"
+rm -f "${COVER_PROFILE}.tmp"
 
 if [[ -n "${REPORT_COVERAGE}" ]]; then
 
```

Comment:
looks like code coverage improvement is the main part of this PR, but it's not actually Not about code coverage : ). From my  pov, code coverage is a serious independent issue that related to the whole project, and nice to have it as separate Task/PR. But it's ok : )